packages/base/room.gts

```diff
@@ -460,8 +461,10 @@ export class RoomCard extends CardDef {
           message: event.content.body,
           formattedMessage,
           index,
-          transactionId: event.unsigned.transaction_id,
+          // These are not guaranteed to exist in the event
+          transactionId: event.unsigned?.transaction_id || null,
```

Comment:
This was causing odd issues elsewhere with uploading cards, some events don't have this when first created.

Reply:
ok--i think i removed all the things depending on this in main so that it is purely informational. are you still seeing other things that consume this value?

Reply:
The old issue was with it being read - I think the last issue I saw was with it being written hence this change.

Reply:
If nothing uses it we can either leave it like this as information or remove it.